internal/cli/install.go

```diff
@@ -110,6 +110,13 @@ var neededOptions = kubernetes.InstallationOptions{
 		Default:     true,
 		Value:       true,
 	},
+	{
+		Name:        "access-control-allow-origin",
+		Description: "Domain allowed to access the API. Comma-separated.",
+		Type:        kubernetes.StringType,
+		Default:     "*", // TODO: Better default than `*`, i.e. allow any ?
```

Comment:
I would say the default should the be most secure. That is the empty list (in which case we shouldn't set the header at all?).

Reply:
Done. (f44ac7992f9bc0ea530703320dee7524cfa51bf7, collapsed history)